test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -3457,6 +3457,62 @@ public async Task GetStatus_WithCondition(bool extendedSessions, string storageP
             }
         }
 
+        [Theory(Skip = "Azure Storage fails due to container deletion")]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        [MemberData(nameof(TestDataGenerator.GetExtendedSessionAndFullFeaturedStorageProviderOptions), MemberType = typeof(TestDataGenerator))]
+        public async Task DurableEntity_GetStatus_WithCondition(bool extendedSessions, string storageProvider, bool fetchInput)
+        {
+            var taskHubName = "GetStatus1";

```

Comment:
Usually we name the task hub after the name of the test, so `nameof(DurableEntity_GetStatus_WithCondition)`. Is there a particular reason you didn't do that here?